bin/tasks/report.js

```diff
@@ -61,30 +63,31 @@ export const generateReport = async (ctx) => {
     .property('buildUrl', build.webUrl)
     .property('storybookUrl', baseStorybookUrl(build.cachedUrl));
 
-  build.snapshots.forEach(({ status, spec, parameters }) => {
+  build.tests.forEach(({ status, result, spec, parameters }) => {
     const suffix = parameters.viewportIsDefault ? '' : ` [${parameters.viewport}px]`;
     const testCase = suite
       .testCase()
       .className(spec.component.name.replace(/[|/]/g, '.')) // transform story path to class path
       .name(`${spec.name}${suffix}`);
 
     switch (status) {
-      case 'SNAPSHOT_ERROR':
+      case 'FAILED':
```

Comment:
You'll also need to change `status` to `status(legacy: false)` in the gql query above I think.